<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLHeightMapWidget</class>
 <widget class="QWidget" name="WLHeightMapWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QCheckBox" name="cbShowGrid">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>show grid</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="cbShow">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>show</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="sbStepInter">
     <property name="prefix">
      <string>step: </string>
     </property>
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cbEnable">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>enable</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="cbType"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
